builders/python-version.psm1

```diff
@@ -0,0 +1,49 @@
+function Convert-Label() {
+    <#
+    .SYNOPSIS
+    Convert generic semver label to native Python label.
+    #>
+
+    param(
+        [Parameter(Mandatory)]
+        [string] $label
+    )
+
+    switch ($label) {
+        "alpha" { return "a" }
+        "beta" { return "b" }
+        "rc" { return "rc" }
+        Default { throw "Invalid version label '$label'" }
+    }
+}
+
+function Convert-Version {
+    <#
+    .SYNOPSIS
+    Convert generic semver version to native Python version.
+    #>
+
+    param(
+        [Parameter(Mandatory)]
+        [semver] $version,
+        [char] $delimiter = "."
+    )
+
+    $majorVersion = $version.Major
+    $minorVersion = $version.Minor
+    $patchVersion = $version.Patch
```

Comment:
Why do we need these three variables?

`$nativeVersion = "{0}.{1}.{2}" -f $version.Major, $version.Minor, $version.Patch`

Reply:
Thanks!